manifests/plugins/haproxy.pp

```diff
@@ -0,0 +1,28 @@
+class collectd::plugins::haproxy (
+  $modules,
+  $filter_metrics = false,
+  $filter_metric_rules = {},
+  $plugin_template = 'collectd/plugins/haproxy/haproxy.conf.erb',
+  $package_name = 'collectd-python',
+  $package_ensure = present,
+  $package_required = false
+) {
+  validate_hash($modules)
+  Exec { path => [ '/bin/', '/sbin/' , '/usr/bin/', '/usr/sbin/' ] }
+  include collectd
+
+  collectd::get_from_github { $title:
```

Comment:
Is there any way to specify a tag to pull from github?  This could cause unexpected changes in customer's environments if we made a new release.